src/API/Plugins.php

```diff
@@ -192,7 +192,12 @@ public function update_item_permissions_check( $request ) {
 	 * @param string $slug The slug of the plugin being installed.
 	 */
 	private function create_install_plugin_error_inbox_notification_for_jetpack_installs( $slug ) {
-		WC_Admin_Notes_Install_JP_And_WCS_Plugins::possibly_add_install_jp_and_wcs_note( $slug );
+		// Exit early if we're not installing the Jetpack or the WooCommerce Services plugins.
+		if ( 'jetpack' !== $slug && 'woocommerce-services' !== $slug ) {
```

Comment:
@becdetat I hope you don't mind; I went back to your original method of checking and returning early here.

I think you were correct about the logic here- we'll just need to follow-up on this to move this check into a filter in onboarding.